source/api_cc/include/DeepPot.h

```diff
@@ -33,6 +33,24 @@ class DeepPot
   * @brief Print the DP summary to the screen.
   * @param[in] pre The prefix to each line.
   **/
+  #if HUAWEI_ASCEND
+  DeepPot  (const std::string & model, const int & nloc, const int & gpu_rank = 0);
+  /**
+  * @brief Initialize the DP.
+  * @param[in] model The name of the frozen model file.
+  * @param[in] gpu_rank The GPU rank. Default is 0.
+  **/
+  void init (const std::string & model, const int & nloc, const int & gpu_rank = 0);
```

Comment:
Do you require the nlocal to be a constant? this is a very strict restriction, as the number of atoms in a local region may change during MD simulations.

Reply:
Yes, the nlocal for Ascend platform is a constant, cause we pad the number of each type of atom. Considering nocal changes during the inference, we increase the value to 1.1 times the original nlocal.